harmoni-web-api/src/main/java/org/harmoniapp/harmoniwebapi/services/TranslationService.java

```diff
@@ -0,0 +1,62 @@
+package org.harmoniapp.harmoniwebapi.services;
+
+import com.google.gson.*;
+import io.github.cdimascio.dotenv.Dotenv;
+import okhttp3.*;
+import lombok.RequiredArgsConstructor;
+import org.harmoniapp.harmoniwebapi.utils.LanguageCodeMapper;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+@ComponentScan(basePackages = {"org.harmoniapp.harmonidata"})
+public class TranslationService {
+
+    private final OkHttpClient client = new OkHttpClient();
+    private static final Dotenv dotenv = Dotenv.configure().filename(".env").load();
+    private final String apiKey =  dotenv.get("API_KEY");
```

Comment:
Instead of 'API_KEY' (environment variable), it is better to use the name of an environment variable that will point to the api in question. In case you need to use another api in the future